client/components/takeQuiz/TakeQuiz.jsx

```diff
@@ -177,12 +185,103 @@ const TakeQuiz = (userId) => {
   }
 
   const submitAnswers = (userScore) => {
-    submitQuizAnswers(14, 1, userScore) // later change to: submitQuizAnswers(quizId, userId, userScore);
+    submitQuizAnswers({
+      correct_answer_count: userScore.correct,
+      incorrect_answer_count: userScore.incorrect,
+      id_quiz: quizId,
+      id_users: 11 // later change to: userId
```

Comment:
Does this still need to use hard coded data?

Reply:
This was before Helen's component to pass down the userId was merged to the branch. Fixing it now